pkg/connection/reconciler.go

```diff
@@ -67,6 +74,10 @@ func MakeReconciler(log logr.Logger, k8sClient client.Client, creator admin.Puls
 		makeNamespacesReconciler(r),
 		makeTopicsReconciler(r),
 		makePermissionsReconciler(r),
+		makeFunctionsReconciler(r),
+		makeSinksReconciler(r),
+		makeSourcesReconciler(r),
+		makePackagesReconciler(r),
```

Comment:
https://github.com/streamnative/pulsar-resources-operator/blob/087c81ebe0c52c38d5e6006f2d761ba87b477b74/pkg/connection/reconciler.go#L176-L181

These reconcilers will execute one by one, one fail will block the flowing reconcilers. So the order is important, I think the function reconciler relies on package reconciler? What will happen if you create function resource first, then create package resource?

Reply:
fix in https://github.com/streamnative/pulsar-resources-operator/pull/216